internal/cli/cmd/login/login.go

```diff
@@ -177,6 +182,11 @@ func (o *loginOptions) run(ctx context.Context) error {
 		refreshToken = ""
 	}
 
+	// To retain the existing user if he's logging into the same server
```

Comment:
Gender-neutral language is more inclusive.

```suggestion
	// To retain the existing user if they're re-logging into the same server
```